src/libs/events.ts

```diff
@@ -312,6 +312,11 @@ export const captureAppcuesEvent = (eventName: string, event: any) => {
     'step_interacted',
     'form_submitted',
     'form_field_submitted',
+    'nps_survey_started',
+    'nps_score',
+    'nps_feedback',
+    'nps_ask_me_later_selected_at',
+    'nps_clicked_update_nps_score',
```

Comment:
I think the nps events technically arent "public-facing-events" so maybe we should create a new list for them? Or we could just rename this list to be something else like "eventNames". The comment above this list should also be updated i think.

Reply:
Yeah makes sense, I'll make a separate list for nps events and update the comment slightly